cfgov/v1/tests/management/commands/test_update_data_snapshot_values.py

```diff
@@ -0,0 +1,37 @@
+from django.core.management import call_command
+from django.test import Client, TestCase
+
+from wagtail.wagtailcore.blocks import StreamValue
+
+from scripts import _atomic_helpers as atomic
+from v1.models.browse_page import BrowsePage
+from v1.tests.wagtail_pages.helpers import publish_page
+
+django_client = Client()
+
+class UpdateDataSnapshotValuesTestCase(TestCase):
+        def test_data_snapshot(self):
+            """ Data Snapshot correctly renders fields on a Browse Page"""
+            browse_page = BrowsePage(
+                title='Browse Page',
+                slug='browse',
+            )
+
+            # Adds a AUT market to a browse page
+            browse_page.content = StreamValue(
+                browse_page.content.stream_block,
+                [atomic.data_snapshot],
+                True
+            )
+            publish_page(child=browse_page)
+
+            # Call management command to update values
+            call_command('update_data_snapshot_values', '--snapshot_file=v1/tests/fixtures/data_snapshots.json')
```

Comment:
@chosak (and anyone else) -- I'd like to get your feedback on this approach for testing this management command.  Does this seem reasonable/ is there a more canonical way to add test fixtures?